taskcluster/ac_taskgraph/loader/build_config.py

```diff
@@ -13,7 +17,113 @@
 from ..build_config import get_components
 
 
+logger = logging.getLogger(__name__)
+
+ALL_COMPONENTS = object()
+# We always build components that have changed files or dependencies, but we also need
+# to rebuild some or all components when other files change. This structure
+# maintains the latter mapping.
+FILE_PATTERNS_TO_AFFECTED_COMPONENTS = {
+    # Just a placeholder example
+    #"^.*changelog.md.*$": ["build-browser-awesomebar"],
+    "^build.gradle$": ALL_COMPONENTS,
+}
+
+def get_files_changed_pr(base_repository, pull_request_number):
+    url = base_repository.replace("github.com", "api.github.com/repos")
+    url += "/pulls/%s/files" % pull_request_number
+
+    r = requests.get(url, timeout=60)
```

Comment:
Followup: Let's retry this call

Reply:
You can use `taskgraph.util.taskcluster.get_session` to get this.